src/display/process_impl/display_proc_impl.cpp

```diff
@@ -6,117 +6,120 @@
 
 #include <memory>
 #include <mutex>
+namespace display {
+  namespace display_process {
```

Comment:
Thinking about how to call classes in this namespace (for example `display::display_process::DisplayProcessImpl()`) makes me feel like we could change this to have less repetition. So I would suggest to change `display_process` to `process`, so we can call `display::process::DisplayProcessImpl()`